ksql-engine/src/main/java/io/confluent/ksql/services/KafkaTopicClientImpl.java

```diff
@@ -104,9 +114,44 @@ public void createTopic(
     }
   }
 
+  /**
+   * We need this method because {@link AdminClient#createTopics(Collection)} does not allow
+   * you to pass in only partitions. Instead, we determine the default number from the cluster
+   * config and then pass that value back.
+   *
+   * @return the default broker configuration
+   */
+  private short getDefaultClusterReplication() {
+    try {
+      final Collection<Node> brokers = adminClient.describeCluster().nodes().get();
```

Comment:
Looks like most of this code is duplicated in `isTopicDeleteEnabled()` in this same file. Can we refactor into a helper method?